packages/auth/amplify_auth_cognito_dart/lib/src/asf/asf_worker.worker.js.dart

```diff
@@ -7,6 +7,7 @@ import 'package:aws_common/aws_common.dart';
 class ASFWorkerImpl extends ASFWorker {
   @override
   String get name => 'ASFWorker';
+
```

Comment:
are the spaces in these worker files changed on purpose?

Reply:
they are autogenerated by worker_bee_builder which uses code_builder to generate dart files. based on code_builder [changelog](https://pub.dev/packages/code_builder/changelog) it adds new line after lambdas.